### ISSUE TITLE
`atom-in-atom` is the only rule with no test pack

### ISSUE THREAD

**Issue by gemshrine** [objectionary/lints#1356]
`atom-in-atom` is the only lint in the repository with no test pack.

Every rule has a directory of its own under `src/test/resources/org/eolang/lints/packs/single/`, named after the stylesheet. Comparing the two sets:

```
rules  = 76   (src/main/resources/org/eolang/lints/**/*.xsl)
packs  = 75   (src/test/resources/org/eolang/lints/packs/single/*)

rules without a pack: atom-in-atom
packs without a rule: none
```

Nothing else covers it either — `atom-in-atom` appears nowhere under `src/test`, and no Java test names it.

The rule is not a trivial one. `src/main/resources/org/eolang/lints/atoms/atom-in-atom.xsl:14`:

```xml
<xsl:for-each select="//o[eo:atom(.) and o[eo:atom(.)]]">
```

It reports an atom that holds another atom, with the message "Atom … may not have any attributes, even if they are atoms, which however exist: …". That predicate has two calls to `eo:atom()` at different depths and a nested-object step between them, which is exactly the shape a pack would pin down: an atom with a plain child should not fire, an atom with an atom child should, and an atom two levels down should be reported against the right element.

Since the whole set is otherwise complete, the gap looks like an oversight when the rule landed rather than a decision.

A pack in the usual shape would close it — a positive case with a nested atom and the expected defect, and a negative case with an atom holding no atoms. Worth adding a check to the build while the gap is fresh: comparing the two directory listings is a few lines and would keep the set complete without anyone having to notice again.

**Comment by yegor256** [comment]
Duplicate of #1307.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by 0crat** [comment]
@gemshrine Thanks for submitting this ticket! However, as you see, it was not accepted by the team, which is sad. You've earned -8 points. By reporting bugs or suggesting enhancements, you help our project improve its quality. Please, don't be discouraged. Instead, try to be more careful next time, and [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) bugs. Your running score is [+3398](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
